apps/judicial-system/web/src/routes/Prosecutor/RestrictionRequest/StepOne/StepOneForm.tsx

```diff
@@ -63,10 +64,14 @@ export const StepOneForm: React.FC<Props> = (props) => {
     defendantId: string,
     updatedDefendant: UpdateDefendant,
   ) => {
-    updateDefendantState(defendantId, updatedDefendant)
+    try {
+      updateDefendantState(defendantId, updatedDefendant)
 
-    if (defendantId) {
-      updateDefendant(workingCase.id, defendantId, updatedDefendant)
+      if (defendantId) {
+        updateDefendant(workingCase.id, defendantId, updatedDefendant)
+      }
+    } catch (error) {
```

Comment:
Consider removing this catch and handle the error in the useDefendant hook in a similar way as is done in the useCase hook.